index.html

```diff
@@ -43,11 +43,108 @@ <h1 class="prof">I am Software Developer</h1>
             <a href="#" class="down-button">
                 <img src="./image/Disabled.svg" alt="">
             </a>
-        
-
+        </div>
+    </section>
 
+     <!-- content 2 -->
+    <section class="main2">
+        <div class="title">
+            <h1>My Recent Work</h1>
```

Comment:
- [ ] let's make sure we use `h2` tag instead of `h1` here so  we follow Semantic HTML.

you could think of `hx` (header tags) as headers for Table of content so normally there is only one h1 (main header for each subject) in our work it's ideally 1 `h1` per page then what comes below should be h2 the sequence goes on you could check [this video](https://www.youtube.com/watch?v=NexL5_Vdoq8&ab_channel=KevinPowell) to learn more about it